src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp

```diff
@@ -398,7 +450,7 @@ void jit_multiply_emitter::emit_isa(const std::vector<size_t>& in_vec_idxs, cons
     VReg src0 = VReg(in_vec_idxs[0]);
     VReg src1 = VReg(in_vec_idxs[1]);
     VReg dst = VReg(out_vec_idxs[0]);
-
+    
```

Comment:
Minor: please remove extra indent

Reply:
Sure, Thank you for the pointers on float implementation, I will go through it & fix the float implementation.

Reply:
done